### PR TITLE
Improve the CheckBox and CheckButton class documentations

### PR DESCRIPTION
**Pull request by Calinou**
This makes the CheckBox and CheckButton classes 100% documented.

### CODE REVIEW

File: doc/classes/CheckBox.xml [hunk 1]
```diff
@@ -1,58 +1,79 @@
...
+		A checkbox allows the user to make a binary choice (choosing only one of two possible options). It's similar to [CheckButton] in functionality, but it has a different apperance. To follow established UX patterns, it's recommended to use CheckBox when toggling it has [b]no[/b] immediate effect on something. For instance, it should be used when toggling it will only do something once a confirmation button is pressed.
 	</description>
 	<tutorials>
 	</tutorials>
 	<methods>
 	</methods>
 	<members>
-		<member name="align" type="int" setter="set_text_align" getter="get_text_align" override="true" enum="Button.TextAlign" default="0" />
-		<member name="toggle_mode" type="bool" setter="set_toggle_mode" getter="is_toggle_mode" override="true" default="true" />
+		<member name="align" type="int" setter="set_text_align" getter="get_text_align" override="true" enum="Button.TextAlign" default="0">
```
Comment (clayjohn) [inline doc/classes/CheckBox.xml]: This and "toggle_mode" are inherited from Button and BaseButton respectively. My understanding was that we weren't going to document inherited members. They are here just to show the different default value. 

However, if I am wrong, please ignore.

Comment (akien-mga) [reply doc/classes/CheckBox.xml]: Indeed, if you run `doctool` this will remove these docs.

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.1.

**Pull request merged**